### PR TITLE
fix: add cppunit library search path

### PR DESCRIPTION
**Pull request by zoyopei**
* **Please check if the PR fulfills these requirements**

- The commit message follows our guidelines
- Tests for the changes have been added (for bug fixes / features)
- Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix.

* **What is the current behavior?** (You can also link to an open issue here)
#1156 

* **What is the new behavior (if this is a feature change)?**
Add subdirectories lib/ and lib64/ to search libraries.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: third-party/cmake/Findcppunit.cmake [hunk 1]
```diff
@@ -34,13 +34,19 @@ find_library(CPPUNIT_LIBRARY_RELEASE
 	NAMES
 	cppunit
 	HINTS
-	"${CPPUNIT_ROOT_DIR}")
+	"${CPPUNIT_ROOT_DIR}"
```
Comment (aceforeverd) [inline third-party/cmake/Findcppunit.cmake:37]: https://cmake.org/cmake/help/latest/command/find_library.html
> Search the paths specified by the HINTS option. These should be paths computed by system introspection, such as a hint provided by the location of another item already found. Hard-coded guesses should be specified with the PATHS option.

or should use `PATHS` instead ?

Cmake manual always hard to understand 😢

Comment (zoyopei) [reply third-party/cmake/Findcppunit.cmake:37]: [https://cmake.org/pipermail/cmake/2010-October/040460.html](url)

> Long story short: Hints get searched before the system paths, and
should only be set by some source of knowledge (location of other
files, etc) and not populated with "guesses" or default locations.
Paths get searched after system locations, and is the place to put
default locations.

I have tested both HINTS and PATHS are ok.

**Comment by aceforeverd** [comment]
Thanks you~. I'll confirm it for other OSs later

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**